addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -227,5 +267,8 @@ func mapAntreaConfigSpec(cluster *clusterv1beta1.Cluster, config *cniv1alpha1.An
 	configSpec.Antrea.AntreaConfigDataValue.FeatureGates.MultiCluster = config.Spec.Antrea.AntreaConfigDataValue.FeatureGates.MultiCluster
 	configSpec.Antrea.AntreaConfigDataValue.FeatureGates.SecondaryNetwork = config.Spec.Antrea.AntreaConfigDataValue.FeatureGates.SecondaryNetwork
 	configSpec.Antrea.AntreaConfigDataValue.FeatureGates.TrafficControl = config.Spec.Antrea.AntreaConfigDataValue.FeatureGates.TrafficControl
+
+	//todo add nsx config here once antreaNsx ia packaged into antrea
```

Comment:
you should also remove antrea Nsx pr and rebase to latest main as antrea Nsx pr is merged into main, I should also add nsx config here